src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -1772,6 +1815,9 @@ impl Step for Assemble {
             if target_compiler.stage == builder.top_stage {
                 builder.info(&format!("Creating a sysroot for stage{stage} compiler (use `rustup toolchain link 'name' build/host/stage{stage}`)", stage=target_compiler.stage));
             }
+
+            maybe_install_llvm_bitcode_linker(target_compiler);
```

Comment:
This means that the bitcode linker will be compiled using the downloaded CI rustc compiler, right?

Reply:
Yes.